Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroLayout/Driver/Controls/Label.cs

```diff
@@ -5,10 +5,14 @@
 /// </summary>
 public class Label : ThemedControl
 {
+    private static Color DefaultTextColor = Color.White;
```

Comment:
I worry about this. Because if its an epaper, text wont show up.

Reply:
Since this isn't really new code, I'd recommend filing an issue

Reply:
it is new code, its green.